src/Pages/Login-Page/login.css

```diff
@@ -0,0 +1,131 @@
+.login-page {
+  font-family: "Montserrat", sans-serif;
+  display: grid;
+  grid-template-rows: 2.5rem minmax(calc(100vh - 7.5rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
+}
+.login__main {
+  padding-top: 3rem;
+  display: flex;
+  justify-content: center;
```

Comment:
your css is also correct , if possible these two property are same as you mentioned in below class (login-form--new-account) you can you that class and remove here